src/Core/Resources/Material.h

```diff
@@ -59,6 +59,7 @@ namespace MxEngine
         float RoughnessFactor = 0.75f;
         float MetallicFactor = 0.0f;
 
+        bool ReceiveSSR = false;
```

Comment:
why its false by default?

Reply:
Because it's in screen space. Most objects even have no enough information on the screen. So I guess users should manually open this for floors or walls etc...